### DISCUSSION TITLE
Welcome to ioBroker.sun2000 Discussions!

### DISCUSSION THREAD

**Discussion by bolliy** [bolliy/ioBroker.sun2000#123] [Ankündigungen]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sun2000-fredi** [comment]
Hi I am using the sun2000 ioBroker Adapter from Bolliy and want to say thank you for that great support & easy using! Yes, I am able to get some data from my very new Inverter Sun2000-12KTL-M5 and from the sDongle, but the data are refreshed around every 20-30 seconds only, and I get a lot of timeout errors reading the 30'000er registers with Modbus TCP reconnections every time. What can I do to solve this issue?
![Issues](https://github.com/user-attachments/assets/d30e4100-0617-4759-a8b0-950a2ca89f0e)
See also my settings:
![settings](https://github.com/user-attachments/assets/fd7efc71-5557-46a0-8dca-fff131559cae)
![Integration](https://github.com/user-attachments/assets/9e76aedf-4b9a-41bc-bc8c-6fe4872a91b6)
![Timing](https://github.com/user-attachments/assets/50730656-561a-4cfb-8407-f488e53e18fa)
Fredi

**Comment by bolliy** [comment]
Normally the first inverter has the modbus id **1** and sometimes the modbus id 16. The modbus id 2 is rather unusual. I think that could be the cause.

**Comment by sun2000-fredi** [comment]
So you mean I have to ask my installer to change the Modbus ID on my inverter from ID 2 to ID 1?

**Comment by bolliy** [comment]
No, does your inverter really have Modbus ID 2?
See the wiki for troubleshooting tips https://github.com/bolliy/ioBroker.sun2000/wiki/Fehlerprotokollierung-und-Fehlerbehebung-(troubleshooting)
Please add the state **sun2000.0.info.JSONhealth**  with the code tag <>.

**Reply by sun2000-fredi** [reply]
Yes, my inverter only "listen" on Port 502 Modbus ID 2. Other IDs such as ID 1 or ID 16 will end in TCP connection refuses.
Unfortunately, I can't add the JSONhealth as I am no longer able to login to ioBroker, since I have updated the Admin Adapter. So I have to solve first the login issue.... (huff...)

**Comment by sun2000-fredi** [comment]
@bolliy should we change my request to the German ioBroker Board? Ich meine den Thread mit dem "langen Bart". Ich spreche Deutsch...
